pyanaconda/ui/cockpit/anaconda-webui/test/ks.cfg

```diff
@@ -0,0 +1,3 @@
+%pre --log=/tmp/kickstart_pre.log
+/usr/libexec/cockpit-ws --no-tls --port 9090 --local-session=cockpit-bridge &
```

Comment:
We might want to do that by default for now. Or we can introduce a new boot option.

Reply:
I don't think we should enable remote access top the Web UI until some sort of authentication is supported. Boot option should be fine though, even if the remote Web UI would not have authentication initially.

Reply:
In any case I think this is a follow-up PR material.

Reply:
Ok, I have created a new issue.